crates/net/eth-wire/src/ethstream.rs

```diff
@@ -87,20 +87,21 @@ where
     /// Handshake with no timeout
     pub async fn handshake_without_timeout<N: NetworkPrimitives>(
         mut self,
-        status: Status,
+        status: UnifiedStatus,
         fork_filter: ForkFilter,
-    ) -> Result<(EthStream<S, N>, Status), EthStreamError> {
+    ) -> Result<(EthStream<S, N>, UnifiedStatus), EthStreamError> {
         trace!(
-            %status,
+            status = %status.into_message(),
             "sending eth status to peer"
         );
-        EthereumEthHandshake(&mut self.inner).eth_handshake(status, fork_filter).await?;
+        let their_status =
+            EthereumEthHandshake(&mut self.inner).eth_handshake(status, fork_filter).await?;
 
         // now we can create the `EthStream` because the peer has successfully completed
         // the handshake
         let stream = EthStream::new(status.version, self.inner);
 
-        Ok((stream, status))
+        Ok((stream, their_status))
```

Comment:
@mattsse, the previous implementation of `handshake_without_timeout` was returning the local status instead of `their_status` in the tuple.  This seems incorrect since downstream functions expect to receive a `their_status` during the handshake flow, so I updated it here.

Reply:
yeah this makes sense